lib/numbers.md

```diff
@@ -5,20 +5,20 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What are the different kinds of numbers in Ruby?
 
-Replace me with your response...
+	The Two types of numbers used regularly are Fixnum (integer) and Float.
 
 # What are some common operations and comparisons you would perform on numbers?
 
-Replace me with your response...
+	Some common comparison operators are ==,!=, <, >, >=, and <=. 
 
 # What is the difference between the `+` operation on a number versus on a String?
 
-Replace me with your response...
+	With numbers the + sign will use math and add the values together while + sign to strings will combine two strings into one string. 
 
 # If you have a _String_ `"20"` and want to perform a mathematical operation (like division or multiplication) on it, will it work? If yes, why? If not, how would you make it work?
 
-Replace me with your response...
+No, the string will need to be converted as an integer such as “20”.to_i
```

Comment:
What result to you get when you run "20" * 3? Does this operation still work, even if it doesn't work the way we'd expect/hope it to?

If you do convert "20" to an Integer, then try to divide it by 3, do you get a precise answer? How might you get a more precise answer?